doc/manuals/admin/upgrading_crossing_2-2-0.md

```diff
@@ -0,0 +1,92 @@
+# Upgrading to 2.2.0 and beyond from a pre-2.2.0 version
+
+Orion 2.2.0 removes the metadata ID feature and this involves a change in DB model. This procedure check if you
+have such metadata ID in your data and, in that case, explains how to deal with that situation.
+
+-   Stop contextBroker
+-   Remove previous contextBroker version
+
+        yum remove contextBroker
+
+-   [Take a backup of your DBs](database_admin.md#backup) (this is just a safety measure in case any problem occurs,
+    e.g some script gets interrupted before finished and your database data ends in an incoherent state)
+-   Download the following script:
+    -   [check_metadata_id.py](https://github.com/telefonicaid/fiware-orion/blob/2.2.0/scripts/managedb/check_metadata_id.py)
```

Comment:
https://github.com/telefonicaid/fiware-orion/blob/2.2.0/scripts/managedb/check_metadata_id.py -> https://github.com/telefonicaid/fiware-orion/blob/2.2.0/scripts/managedb/upgrade-2.2.0/check_metadata_id.py 

@fisuda please sync in PR #3436

Reply:
Fixed in c21dd86